css/intro.css

```diff
@@ -0,0 +1,98 @@
+:root {
+  --color-toast: #ff0000;
+}
+
+.input-label-dark {
+  margin-left: 0.5rem;
+}
+
+.input-big {
+  height: 10.25rem;
+}
+
+.loading {
+  margin: 20rem auto;
+  text-align: center;
+}
+
+.not-found,
+.not-authorized {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  min-height: 70vh;
+  margin: 3rem 0;
+}
+
+.not-found-text,
+.not-authorized-text {
+  font-size: 1.75rem;
+  text-align: center;
+}
+
+.toast-box {
+  position: absolute;
+  top: 2rem;
+  right: 2rem;
+  display: flex;
+  align-items: flex-end;
+  overflow: hidden;
+  padding: 1rem;
+}
+
+.toast {
+  width: 18rem;
+  height: 3rem;
+  background: var(--color-bg);
+  color: var(--color-toast);
+  font-weight: 500;
+  margin: 1rem 0;
+  box-shadow: 0 0 20px rgba(0, 0, 0, 0.3);
+  display: flex;
+  align-items: center;
+  position: relative;
+}
+
+.toast .icon {
+  width: 1.5rem;
+  margin: 0 1rem;
+}
+
+.toast::after {
+  content: '';
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 0.3rem;
+  background: var(--color-toast);
+  animation: anim 5s linear forwards;
+}
+
+@keyframes anim {
+  100% {
+    width: 0;
+  }
+}
+
+.hidden {
+  display: none;
+}
+
+@media screen and (max-width: 500px) {
+  .user-input {
+    font-size: 1rem;
+    width: 22rem;
+  }
+
+  .not-found-text,
+  .not-authorized-text {
+    font-size: 1rem;
```

Comment:
we shouldn't use rem for font-size I suggest you read on this

Reply:
read and ask which will be better, not just because the reviewer said so, for fonts we should use em

Reply:
Thank you @bhtibrewal for pointing me to read about which unit we have to use for font-size
As I read `em` is relative to parent element and `rem` is relative to html element 
It good to use rem for font size 
when i read this two article- 
1. https://www.freecodecamp.org/news/why-use-rem-to-set-font-size-in-css/ 
2. https://medium.com/@dixita0607/css-units-for-font-size-px-em-rem-79f7e592bb97

Reply:
understanding the meaning and use of different units is important

Reply:
okk @bhtibrewal thank you...I will try to careful from next time